enhancements/csvless-bundles.md

```diff
@@ -176,56 +483,106 @@ Security impacts are minimal because the number of kubernetes objects that OLM s
 
 ### Implementation Details
 
-#### Required Manifests
-
-The absence of a CSV means that there is a set of manifests that must ALWAYS be present to build `registry+v2` bundles. These manifests are ALWAYS __required__:
-
-- a `Deployment` for the operator
-- a `metadata/olm.yaml` file with the following fields
-  - name
-  - version
-  - minKubeVersion
-  - installModes
 
-In certain situations, more manifests are required:
+#### Registry Database Schema Changes
 
-- providing an API from an APIService requires
-  - an `APIService`
-  - the `Service` it references
-- providing an admission webhook
-  - a `MutatingWebhookConfiguration` or a `ValidatingAdmissionWebhook`
-  - the `Service` it references
+In order to support adding `registry+v2` bundles to a registry database, the `operatorbundle` table schema must be updated to enable more than one mediatype. The table will be extended with a new `mediatype` column with a value that can be one of `{registry+v1, registry+v2}`. A respective migration will also be added to [operator-registy](https://github.com/operator-framework/operator-registry/tree/master/pkg/sqlite/migrations) to cover this schema change.
```

Comment:
> Does this mean that databases created w/ a newer version of OPM will not function with older versions of the server?

No, adding a field is generally always backwards compatible. What it does mean, however, is that the `registry+v2` bundles in an index used on an older cluster will not be able to unpack them. The behavior in this scenario is described in the _Registry API Changes_ section immediately below.

> If this is the case, we need a mitigation strategy for the pipeline.

I imagined that this would go into a pipeline enhancement proposal for _using_ `registry+v2` bundles. But now that you mention it, I'll add something that warns of mixing mediatypes in indexes bound for clusters with incompatible OLM versions.